common/djangoapps/nodebb/signals/handlers.py

```diff
@@ -14,12 +14,13 @@
 @receiver(post_save, sender=CourseOverview, dispatch_uid="nodebb.signals.handlers.create_category_on_nodebb")
 def create_category_on_nodebb(sender, instance, created, **kwargs):
     if created:
-        status_code, response_body = NodeBBClient().categories.create(name=instance.display_name)
+        community_name = '%s- %s- %s- %s' % (instance.display_name, instance.id.org, instance.id.course, instance.id.run)
```

Comment:
Please remove spaces from community_name pattern, make it like this `'%s-%s-%s-%s'` and replace existing spaces in instance.display_name, instance.id.org, instance.id.course, instance.id.run with `_`